.travis.yml

```diff
@@ -16,11 +19,18 @@ matrix:
     - os: linux
       dist: precise
       jdk: openjdk7
+      before_install:
+        # Fix buffer overflow in getLocalHostName of OpenJDK 7 by shortening the host name
+        - cat /etc/hosts
+        - sudo hostname "$(hostname | cut -c1-63)"
+        - sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts > /tmp/hosts
```

Comment:
Should ".*" be outside the "\\)"?

Reply:
Regex is from https://github.com/travis-ci/travis-ci/issues/5227#issuecomment-165131913
I believe that `.*` is intentional to allow for a range of possible IP addresses.

Reply:
Yes, makes sense that the new hostname is simply being appending rather than replacing the longer hostname that may exist. :+1: